src/main/java/datawave/query/util/MetadataHelper.java

```diff
@@ -1126,6 +1126,44 @@ public long getCardinalityForField(String fieldName, String datatype, Date begin
         return count;
     }
     
+    public Date getEndDateCap(Date begin, Date end) throws TableNotFoundException {
+        
+        // Get all the rows in DatawaveMetadata for the field, only in the 'f'
```

Comment:
Where are the options for scanning the 'f' column only?  I must not have had enough coffee because I'm not seeing it.